src/FakeItEasy/Creation/CastleDynamicProxy/CastleDynamicProxyInterceptionValidator.cs

```diff
@@ -46,7 +46,7 @@ private static string GetReasonForWhyMethodCanNotBeIntercepted(MethodInfo method
                 }
                 else
                 {
-                    return "Static methods can not be intercepted.";
+                    return "Non-virtual methods can not be intercepted. Virtual methods are methods explicitly marked virtual, overriding methods, abstract methods, and interface methods.";

```

Comment:
I would be inclined to leave this message alone. Unlike the "sealed/non-virtual" confusion, the `static` keyword is right in the code when a static method is defined, so there should be no confusion. Also, if the two messages (here and on line 37) are the same, it makes it harder to tell exactly what went wrong.